Doc/library/importlib.rst

```diff
@@ -416,12 +416,16 @@ ABC hierarchy::
         Loaders that have a file-like storage back-end
         that allows storing arbitrary data
         can implement this abstract method to give direct access
-        to the data stored. :exc:`OSError` is to be raised if the *path* cannot
-        be found. The *path* is expected to be constructed using a module's
-        :attr:`__file__` attribute or an item from a package's :attr:`__path__`.
+        to the data stored.
+
+        An :exc:`OSError` is to be raised if the *path* cannot be found, and
```

Comment:
I understand that you have split this in a separate paragraph because the text about exceptions is now larger, but "`The *path* is expected`" does not fit here. It should be either split in a separate paragraph (but then we will have too small paragraphs) or merged with the previous paragraph.

Reply:
How about putting " The
        *path* is expected to be constructed using a module's :attr:`__file__`
        attribute or an item from a package's :attr:`__path__`." after "to the data stored" and have a single paragraph about exceptions?

Reply:
Yes, this is what I suggested. Sorry for not been clear.

Reply:
I've also simplified the other occurrences. No need for telling the reason why we would have a ValueError IMO, it's sufficient to know that the exception can be raised.